uapi/uapi/uapi.rs

```diff
@@ -57,6 +142,41 @@ pub fn get_cycles() -> Result<u64, Status> {
     Ok(result)
 }
 
+
+/// Get back the task handle associated to the input task label
+///
+/// # Usage
+///
+/// While task labels are compile-time fixed and known by tasks that communicate
+/// with each other, the label can't be used at runtime in order to use the
+/// Sentry communication API.
+///
+/// Instead, at each job creation (including recreation when a job exists and restart),
+/// a job handle is forged dynamically. This API allows any tasks that know
+/// the corresponding task label to get back the associated handle.
+///
+/// # Example
+///
+/// ```rust
+/// let label = 0xbabe;
+/// let handle = get_process_handle(label);
+/// match handle {
+///   taskh_t => (),
+///   any_err => Err(any_err),
+/// };
+/// ```
+///
+pub fn get_process_handle(process: ProcessLabel, _handle: *mut taskh_t) -> Result<taskh_t,Status> {
+    match syscall::sys_get_process_handle(process) {
+        Status::Ok => (),
+        any_err => return Err(any_err),
+    };
+    // FIXME: we might use copy_to_user here
+    let exch_area = unsafe { &mut SVC_EXCHANGE_AREA[..4] };
+    let result = u32::from_ne_bytes(exch_area.try_into().map_err(|_| Status::Invalid)?);
+    Ok(result)
+}
```

Comment:
`_handle` is not used

Reply:
Right. Will be removed that, as, as described in the UAPI page of the wiki, this part will be pushed up to a dedicated crate.